app/src/main/res/values/strings.xml

```diff
@@ -163,5 +163,8 @@
     <string name="title_activity_login_facebook">LoginFacebookActivity</string>
     <string name="submit_your_feedback">Submit your Feedback</string>
     <string name="ratings_empty">There is no ratings yet.</string>
+    <string name="create_form_upload_image">Upload image</string>
```

Comment:
Maybe it would be good to move these strings in the create_event_ string block instead of mixing them with feedback strings

Reply:
You're right ! Will do it right now !